### ISSUE TITLE
USER ISSUE: Issues staying on server - constant kick out

### ISSUE THREAD

**Issue by Fazamere** [StrangeLoopGames/EcoIssues#5804]
**Version:** 0.7.2.3 beta

**Steps to Reproduce:**
Trying to logon repeatedly on same server (US based) from Australia
Everytime I logon, it loses responsiveness after a few seconds
Latest time, I saw my own character in front of me when I relogged in.

**Expected behavior:**
Logged me on and stayed logged in or said something like weak connection.

**Actual behavior:**
The following error message popped up:
```
Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Value cannot be null.
Parameter name: item


Stack:
   at System.Collections.Immutable.Requires.FailArgumentNullException(String parameterName)
   at System.Collections.Immutable.ImmutableHashSet`1.Contains(T item)
   at Eco.Core.Controller.ControllerInstanceInfo.TryBind(INetClient client)
   at Eco.Core.Controller.ControllerManager.TryBindController(IController controller, INetClient boundClient)
   at Eco.Core.Controller.ControllerManager.PackageController(IController controller, INetClient boundClient)
   at Eco.Gameplay.Objects.WorldObject.GetBSON(InteractionContext context)
   at Eco.Gameplay.Objects.WorldObject.OnActInteract(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Netw

**Comment by MarjieAVolk** [comment]
Similar to https://github.com/StrangeLoopGames/EcoIssues/issues/5328, https://github.com/StrangeLoopGames/EcoIssues/issues/5628, https://github.com/StrangeLoopGames/EcoIssues/issues/5789

**Comment by darkshad9999** [comment]
Software: Eco Client 0.7.2.4 Beta (64 bit) https://s3-us-west-2.amazonaws.com/eco-releases/EcoPC_v0.7.2.4-beta.zip
OS/Web Browser? Windows 7 (64 bit)

Detailed Description of Issue:

Well I got this when I was trying to collect my fishing traps I was under water and was trying to
pick up my fishing trap. I was down under the water and was trying to pick up my fishing
trap and I got this error. I was located at 526,72 on the map.

This was happening on March 14 2018 at about 4:15 PM MST (GMT-0700)

Reproducible? How difficult is it to reproduce?: Unknown I only got this error once.

Steps to reproduce bug/issue: Unknown only got this once.

N/A I am not sure how to reproduce it at this time. I tried but it didn't reproduce.

Additional Comments:

Can some one please take a look at this error the full error that I got was as follows

Caught exception:

<size=60.00%>Caught exception invoking RPC PlayerInteract on Player!

Value cannot be null.
Parameter name: item


Stack:
   at System.Collections.Immutable.Requires.FailArgumentNullException(String parameterName)
   at System.Collections.Immutable.ImmutableHashSet`1.Contains(T item)
   at Eco.Core.Controller.ControllerInstanceInfo.TryBind(INetClient client)
   at Eco.Core.Controller.ControllerManager.TryBindController(IController controller, INetClient boundClient)
   at Eco.Core.Controller.ControllerManager.PackageController(IController controller, INetClient boundClient)
   at Eco.Gameplay.Objects.WorldObject.GetBSON(InteractionContext context)
   at Eco.Gameplay.Objects.WorldObject.OnActInteract(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)



Stack:
   at Eco.Shared.Networking.RPCManager.TryInvoke(Object controller, String methodname, BSONObject bsonArgs, Object& result)
   at Eco.Shared.Networking.RPCManager.InvokeOn(BSONObject bson, Object controller, String methodname)
   at Eco.Shared.Networking.RPCManager.HandleReceiveRPC(INetClient client, BSONObject bson)
   at Eco.Shared.Networking.NetPort.ProcessBuffer()
   at Eco.Shared.Networking.NetPort.Update()
   at Eco.Plugins.Networking.Clients.Client.<.ctor>b__56_0()
</size>

A screenshot of this error can be seen at http://ibb.co/kZmrHx

**Comment by metachronism** [comment]
should be fixed as of 7.3
